dear_petition/petition/resources.py

```diff
@@ -73,4 +172,289 @@ def skip_row(self, instance, original, row, import_validation_errors=None):
+
+    def dehydrate_jurisdiction(self, record):
+        jurisdiction = getattr(record, 'jurisdiction')
+        return constants.JURISDICTION_MAP[jurisdiction]
+    
+    def hydrate_jurisdiction(self, field, data):
+        attribute = field.attribute
+        for k,v in constants.JURISDICTION_MAP.items():
+            if v == data[attribute]:
+                data[attribute] = k
+                break
+
+        return data
+
+    class Meta:
+        model = models.Offense
+        exclude = ('id', 'ciprs_record')
+        export_order = ('ciprs_record_id',)
+        force_init_instance = True
+
+
+class OffenseRecordResource(MultiModelResource):
+
+    NUM_INDENT=1
+
+    offense_id = fields.Field(attribute="offense_id")
+    count = ExcelField(attribute='count', column_name='#')
+    severity = ExcelField(attribute='severity', dropdown=DataValidation(type="list", formula1=f'"{",".join(constants.SEVERITIES._db_values)}"', showDropDown=True))
+    action = ExcelField(attribute='action', dropdown=DataValidation(type="list", formula1=f'"{",".join(constants.ACTIONS._db_values)}"', showDropDown=True))
+
+    def get_export_fields(self):
+        return [field for field in super().get_export_fields() if field.column_name != 'offense_id']
+    
+    def get_export_headers(self):
+        return [header for header in super().get_export_headers() if header not in ('Offense Id')]
+
+    class Meta:
+        model = models.OffenseRecord
+        exclude = ('id', 'agency', 'offense')
+        export_order = ('offense_id',)
+        force_init_instance = True
+
+class ClientResource(MultiModelResource):
+    dob = ExcelField(attribute='dob', column_name='Date of Birth')
+
+    class Meta:
+        model = models.Client
+        exclude = ('id', 'contact_ptr', 'category','user')
+        force_init_instance = True
+
+    def get_import_fields(self):
+        return ["batch_id"] + super().get_import_fields()
+
+class RecordSummaryResource(resources.ModelResource):
+
+    class Meta:
+        model = models.Batch
+
+    def __init__(self):
+        super().__init__()
+        self.batch_resource = BatchResource()
+        self.client_resource = ClientResource()
+        self.record_resource = RecordResource()
+        self.record_headers = self.record_resource.get_export_headers()
+        self.first_record_header = self.record_headers[0]
+        self.offense_resource = OffenseResource()
+        self.offense_headers = self.offense_resource.get_export_headers()
+        self.first_offense_header = self.offense_headers[0]
+        self.offense_record_resource = OffenseRecordResource()
+        self.offense_record_headers = self.offense_record_resource.get_export_headers()
+        self.first_offense_record_header = self.offense_record_headers[0]
+
+    def export(self, batch_object=None, *args, **kwargs):
+        # Initialize a new dataset with headers
+        assert isinstance(batch_object,models.Batch), "Queryset must be a Batch object"
```

Comment:
Better to raise error.  I believe asserts can be disabled through configuration.